rocky/reports/report_types/systems_report/report.html

```diff
@@ -2,7 +2,7 @@
 
 <div class="horizontal-scroll">
     <table>
-        <caption class="visually-hidden">{% translate "Overzicht Handmatig gevrijwaarde gescande assets:" %}</caption>
+        <caption class="visually-hidden">{% translate "Declared scanned assets:" %}</caption>
```

Comment:
This should just say:

"Selected assets", as its not limited to just declared assets if the user makes a different choice earlier on in the flow.